lib/roo/xls/excel.rb

```diff
@@ -13,6 +14,8 @@ class Excel < Roo::Base
       rescue LoadError
         false
       end
+    INTEGER = 1.class
+    BIGNUM = (1**60).class
```

Comment:
We should support what's below without these definitions. Anything that `is_a?(Integer)` will apply to Fixnum, Bignum, and Integer classes.